package.json

```diff
@@ -16,7 +16,8 @@
     "publish:ci-prod": "pnpm publish && pnpm exec changeset tag && git push --follow-tags",
     "publish:ci-dev": "pnpm exec changeset version --snapshot pr && pnpm publish --tag dev --no-git-checks",
     "lint-staged": "lint-staged",
-    "test:integration": "INTEGRATION=1 vitest run test/integration"
+    "test:integration": "INTEGRATION=1 vitest run test/integration",
+    "test:integration-sentinel": "SENTINEL_INTEGRATION=1 vitest run test/integration"
```

Comment:
I'm wondering if this is actually good pattern to use envs here. Maybe we should utilize some tags/filtering/vitest projects for that to selectively run them? If we have dedicated Vitest project, we could even start docker containers automatically I believe

https://vitest.dev/guide/filtering

@krzysztofzuraw wdyt?

Reply:
Yes - I think we can have either folder with `test/integration` or name of the file `test-name.integration.ts`.

By the way - what about running both integration suites (redis / sentinel) in the same `test:integration`?

Reply:
I think once we have projects, we can select which projects to run, and we can run both integration suites with single scriot

Reply:
we talked with @lkostrowski and decided that we can move right now with envs in this PR so we wont end up changing how to run tests in PR that adds IORedis 😄